src/mongocxx/test/collection.cpp

```diff
@@ -534,21 +534,25 @@ TEST_CASE("CRUD functionality", "[driver::collection]") {
 
         // copy into a vector.
         std::vector<bsoncxx::document::value> results;
+        
         for (auto&& result : distinct_results) {
```

Comment:
@tessavitabile - Yeah and the logs don't show much. If you can't repro locally, I'm inclined to move forward with the patch. I don't see how they are connected.